src/react/components/TimeSeries/TimeSeriesSlider.jsx

```diff
@@ -1,21 +1,24 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
-import { playbackSpeedOptions, sliderTypes, timeUnits } from './util/constants';
+import { sliderTypes, timeUnits } from './util/constants';
 import { getDifferenceCalculator, calculateSvgX } from './util/calculation';
+import { getHeaderTheme } from 'oskari-ui/theme/ThemeHelper';
```

Comment:
I think we want to use the `getNavigationTheme()` instead as it it uses the button colors that matches the current colors on this controls more closely than the "popup colors" that `getHeaderTheme()` is used for.